frontend/docs/pages/home/features/cancellation.mdx

```diff
@@ -1,16 +1,45 @@
+import { Tabs } from "nextra/components";
+
 # Cancellation in Hatchet Workflows
 
 Hatchet provides a mechanism for canceling workflow executions gracefully, allowing you to stop running workflows and their associated steps when needed. Cancellation can be triggered on graceful termination of a worker or automatically through concurrency control strategies like [`CANCEL_IN_PROGRESS`](./concurrency/cancel-in-progress), which cancels currently running workflow instances to free up slots for new instances when the concurrency limit is reached.
 
-## Cancellation Mechanism in Node.js
+When a workflow is canceled, Hatchet sends a cancellation signal to all the currently executing steps. The steps can then check for the cancellation signal and take appropriate action, such as cleaning up resources, aborting network requests, or gracefully terminating their execution.
 
-{/* TODO Python, GO */}
+{/* TODO Python */}
 
-Hatchet uses the standard `AbortController` and `AbortSignal` interfaces from Node.js to handle cancellation. Each step in a workflow has access to a `context.controller` property, which is an instance of `AbortController`. The `AbortController` provides a way to signal cancellation to the step and any asynchronous operations it may be performing.
+## Cancellation Mechanism in Go
```

Comment:
I think it'd be great to have this in a set of tab group using `<Tabs>`

Reply:
Yeah that makes sense actually, will make the changes